src/views/BridesmaidsGroomsmen.js

```diff
@@ -0,0 +1,167 @@
+import React, { Component } from "react";
+import styled from "@emotion/styled";
+
+import { AppContext } from "../AppProvider";
+import Main from "../components/Main";
+import Card from "../components/Card";
+
+const StyledPageHeaderImage = styled.div`
+  position: absolute;
+  width: 100%;
+  height: 25.125em;
+  background-image: ${props => `url(${props.src})` || "none"};
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+`;
+
+const StyledPageTitle = styled.h1`
+  font-size: 2rem;
+  position: absolute;
+  left: 50%;
+  bottom: 0.8em;
+  padding: 0.5em 1em;
+  letter-spacing: 0.083em;
+  background-color: rgba(255, 255, 255, 1);
+  color: rgba(54, 63, 84, 1);
+  transform: translate3d(-50%, 50%, 0);
+  box-shadow: 0 5px 10px 0 rgba(37, 41, 82, 0.1);
+`;
+
+const StyledSection = styled.section`
+  display: flex;
+`;
+
+const StyledCardContainer = styled.article`
+  width: 50%;
+  padding: 31.25em 1.5em 7em 1.5em;
+  background-color: ${props => props.color};
+`;
+
+const StyledContainerHeading = styled.h2`
+  font-size: 4.375rem;
+  font-weight: bold;
+  color: ${props => props.color};
+  text-align: center;
+  margin-bottom: 2em;
+`;
+
+const StyledCardRow = styled.div`
+  display: flex;
+  justify-content: space-around;
+`;
+
+const StyledSpecialTitle = styled.div`
+  position: absolute;
+  font-size: 1.25rem;
+  top: -2em;
+  left: 50%;
+  transform: translate3d(-50%, 0, 0);
+  white-space: nowrap;
+  color: ${props => props.color};
+  display: flex;
+  align-items: center;
+`;
+
+const StyledCrown = styled.div`
+  height: 1.75em;
+  width: 1.75em;
+  background-image: ${props => `url(${props.src})` || "none"};
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  margin: 0 0.5em;
+`;
+
+const bridesmaids = [
+  [{ name: "Michelle", maidOfHonor: true }],
+  [{ name: "Shirley" }, { name: "Alexia" }],
+  [{ name: "Tracy" }, { name: "Cathy" }],
+  [{ name: "Priscilla" }]
+];
+
+const groomsmen = [
+  [{ name: "Richard", bestMan: true }],
+  [{ name: "Mario" }, { name: "Tommy" }],
+  [{ name: "David" }, { name: "Jackson" }],
+  [{ name: "Ken" }, { name: "Matthew" }]
+];
+
+class BridesmaidsGroomsmen extends Component {
+  componentDidMount() {
+    const { handleIsBrandDark, handleIsNavbarDark } = this.props;
+    handleIsBrandDark(false);
+    handleIsNavbarDark(false);
+  }
+
+  render() {
+    return (
+      <Main>
+        <StyledPageHeaderImage src="/img/photos/bridesmaids_groomsmen_header.png">
+          <StyledPageTitle>MEET THE...</StyledPageTitle>
+        </StyledPageHeaderImage>
+        <StyledSection>
+          <StyledCardContainer color="rgba(246, 233, 203, 1)">
```

Comment:
The code for the bridesmaids is somewhat similar compared to the groomsmen. The card container and its content could be abstracted into a component and sent props to configure it accordingly.